src/test/runTest.ts

```diff
@@ -41,15 +41,24 @@ async function main(): Promise<void> {
     // Download VS Code, unzip it and run the integration test
     // start in the fixtures folder to prevent the language server from walking all the
     // project root folders, like node_modules
-    console.log('_______________LATEST_____________________');
-    await runTests(options);
-
-    console.log('_______________INSIDERS___________________');
-    options.version = 'insiders';
-    await runTests(options);
+    const vscodeVersion = process.env['VSCODE_VERSION'];
+    switch (vscodeVersion) {
```

Comment:
What do you think of removing this `switch` statement entirely? With the new matrix, it's clear which version is being tested. We could then add `version` to the `TestOptions` object in line 34.

If we want to keep some output, we could do something like this:
```ts
console.log(`_______________${options.version ?? 'latest'}_____________________`);
```